packages/types/src/git.ts

```diff
@@ -0,0 +1 @@
+export const gitStatusType = ['modified', 'created', 'unmodified'] as const
```

Comment:
It's almost like we should have this model of files for everything in the `constants.ts` file.

I like this though.